### ISSUE TITLE
Error_Code_IO_Bad_HTTP_Status

### ISSUE THREAD

**Issue by ranjunair** [mhdzumair/MediaFusion#337]
I get this error on a few films and series under media fusion add on Stremio. How can I fix this?

**Comment by mhdzumair** [comment]
Hi @ranjunair this is a generic error message. It's not possible to identify the root cause. Could you provide more information? like which device, at what time this issue occurred like is it on installing the addon or opening a movie etc

**Comment by ranjunair** [comment]
It happens on a firestick, and when I try to open a Malayalam movie or series. Would you like more specifics?

**Comment by mhdzumair** [comment]
Did you getting this error always? Or sometimes occurs?

**Comment by ranjunair** [comment]
Yes, I get this error always when I play a certain show. It’s a Malayalam series - Jai Mahendran
